src/transformers/tools/base.py

```diff
@@ -599,6 +599,10 @@ def fn(*args, **kwargs):
 
 # TODO: Migrate to Accelerate for this once `PartialState.default_device` makes its way into a release.
 def get_default_device():
+    logger.warning(
+        "`get_default_device` is deprecated and will be replaced with Accelerate's `PartialState.defualt_device` "
```

Comment:
```suggestion
        "`get_default_device` is deprecated and will be replaced with `accelerate`'s `PartialState().device` "
```

Let's use the right API please :)